src/tetris/player/player.ts

```diff
@@ -0,0 +1,46 @@
+import Field from 'tetris/field/field';
+
+export default abstract class Player {
+
+	//region public members
+	//endregion
+
+	//region public methods
+	public abstract update(time: number, delta: number): void;
+	//endregion
+
+	//region constructor
+	protected constructor(field: Field) {
+		this._field = field;
+	}
+	//endregion
+
+	//region private members
+	private _field: Field;
```

Comment:
This should be either `protected` or there should be a protected getter.

Reply:
Discussed it offline. Seems to be fine so far.